content/posts/annoucing-slight-v0_3.md

```diff
@@ -0,0 +1,226 @@
+---
+title: "Annoucing Slight v0.3"
+description: "Announcing new features and updates on Slight v0.3 release."
+date: 2023-02-25 00:00:00 +0000 UTC
+authorname: "Jiaxiao (Joe) Zhou"
+author: "@jiaxiao_zhou"
+authorlink: "https://twitter.com/jiaiao_zhou"
+tags: ["slight", "spiderlightning", "Wasm", "WASI"]
+---
+
+We are excited to announce the release of Slight v0.3, the latest version of our WebAssembly runtime that utilizes SpiderLightning (also known as wasi-cloud) capabilities.
+
```

Comment:
OK, here we need a bullet summary of the new things that you discuss below. Fill the reader's mind _instantly_ with the magic that might interest them -- so that they keep reading. The way this reads, they should just magically be in love with Slight, and while we wish that were the case sadly experience tells us that they are not. 

What they ARE interested in is what they can DO with this -- but their reading time is precious and short. Tell them IMMEDIATELY what magic this contains right up front. Something like:

New in this release:
1. Making outbound http calls to services
2. working with message queues and pubsub
3. working with sql databases

and more is on the way. Let's dive into the new features.